### PR TITLE
Clean up admin ports doc

### PR DESCRIPTION
**Pull request by branden**
This PR removes an irrelevant port and applies consistent formatting and naming.

# Issues

N/A

# Checklist

 - [ ] Included a test which will fail if code is reverted but test is not
 - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

**Comment by branden** [comment]
Pinging @sargun for review.

**Comment by branden** [comment]
@sargun which systemd unit provides "DCOS Overlay" (port 64000)?

File: docs/admin-ports.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 61421: dcos-minuteman
  - 62053: dcos-spartan
  - 62080: dcos-navstar
- - 62501 - Spartan
- - 62502 - Navstar
- - 62503 - Minuteman
+ - 62501: dcos-spartan
+ - 62502: dcos-navstar
+ - 62503: dcos-minuteman
```
Comment (sargun) [inline docs/admin-ports.md]: This is a dup?

File: docs/admin-ports.md [hunk 2]
```diff
@@ -38,7 +38,6 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
- - 62500: Networking API
```
Comment (branden) [inline docs/admin-ports.md]: I don't have a good answer for that. I could copy the doc and keep this port, but without a process to sync that version with this one, it'll just go out of date.

File: docs/admin-ports.md (hunk 1 above)
Comment (sargun) [inline docs/admin-ports.md]: Ah.

Comment (sargun) [inline docs/admin-ports.md]: 👍

**Comment by sargun** [comment]
The kernel uses it, no systemd unit.

File: docs/admin-ports.md (hunk 2 above)
Comment (alberts) [inline docs/admin-ports.md]: @branden is going to add a doc for just for enterprise-only ports

**Pull request merged**